src/main/java/newbank/server/NewBank.java

```diff
@@ -66,56 +83,67 @@ public synchronized CustomerID checkLogInDetails(final String username, final St
     return null;
   }
 
+  /**
+   * Retrieve and display account information for a given customer
+   *
+   * @param customerID The customer identifier
+   * @return account information
+   */
+  public synchronized String showAccountsFor(final CustomerID customerID) {
+    Customer customer = customers.get(customerID.getKey());
+
+    return showAccountsFor(customer);
+  }
+
+  private String showAccountsFor(final Customer customer) {
```

Comment:
Is there any value of this overload? Not sure if we need both methods.

Reply:
Agreed that the overloads need re-visiting at some point. 

Kindest Regards,
Jules